src/components/About/Directors.module.css

```diff
@@ -0,0 +1,82 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+*, *::before, *::after {
+  box-sizing: border-box;
+}
+
+.container {
+  background-color: var(--deep-jungle-blue);
+  display: flex;
+  position: relative;
+  justify-content: center;
+  z-index: 1;
+}
+
+.topImage {
+  width: 200px;
+  height: 200px;
+  position: absolute;
+  top: -100px;
+  left: -100px;
+}
+
+.bottomImage {
+  width: 147px;
+  height: 100px;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+
+.article {
+  margin-top: 5.5rem;
+  margin-bottom: 5.5rem;
+}
+
+.title {
+  color: var(--primary-white);
+  text-align: center;
+}
+
+.profiles {
+  margin-top: 3rem;
+  display: grid;
+  grid-template-rows: repeat(6, 1fr);
+  row-gap: 1.5rem;
+}
+
+@media (min-width: 768px) {
+  .topImage {
+    top: 0;
+  }
+
+  .article {
+    margin-top: 6.25rem;
+    margin-bottom: 6.25rem;
+  }
+
+  .profiles {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    column-gap: 0.688rem;
+    row-gap: 3rem;
+  }
+}
+
+@media (min-width: 960px) {
```

Comment:
I was just curious as to why we use 960 instead of 1440? The desktop design seems to be made for 1440 so parts of the left and right profile cards are hidden at 960.

Reply:
There are a variety of approaches to where breakpoints are set, but the important thing is to remember that most views in a category will not be at exactly the size shown on the design. In the case of desktops, 1440px is both quite a bit narrower than most desktop (and laptop) displays, and quite a bit wider than most tablet displays. Remember that 1080px is where HD starts, so that's already quite wide! 

960px is larger than all the common tablets, so it's a reasonable (and common) point to choose for the desktop breakpoint. You'll note that the Figma designs are at 1440px wide (desktop), 768px wide (tablet), and 375px wide (mobile). We have to be aware of sizes in between all of those—and occasionally larger than 1440px, though not in this design.

There are multiple approaches to how to fix the display of the cards at 960px width. One would be to make the cards more responsive; another would be to make the layout of the cards more fluid so it automatically wraps to two cards per line if there's overflow.

Reply:
Thanks for the explanation! I've changed the cards to be displayed as flex instead of grid so that the cards wrap. The row/column gaps between the cards differ proportionately for the design sizes so I'm not sure what they should be for intermittent sizes. Another weird thing is that the card width in the design is smallest at 768px so I can't even scale the cards as the width increases. However, I've ensured that the number of cards per row consistently increases with width.